manifests/template-example.yml.template

```diff
@@ -16,6 +16,9 @@ additional-emails: []
 #            the xpi package.json lives. Optional. Defaults to the
 #            root of the repository.
 directory: one
+# string: docker-image. Optional. Defaults to node-14 as found
+#         in taskcluster/ci/[build|test]/kind.yml .
+docker-image: node-14
```

Comment:
Hm, I'm trying to figure out why this and the below changes are showing up here.
Didn't we merge these in #111? I see the changes in https://github.com/mozilla-extensions/xpi-manifest/blob/master/manifests/template-example.yml.template#L19-L21